### PR TITLE
chore: move import map contents into deno.jsonc

### PR DESCRIPTION
**Pull request by harrysolovay**
Now possible thanks to [Deno v1.30.0](https://github.com/denoland/deno/releases/tag/v1.30.0).

### CODE REVIEW

File: examples/smart_contract.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as C from "capi/mod.ts"
 
-import { client } from "capi/frame/zombienet/examples/smart_contract/zombienet.toml/collator01/@v0.9.360/mod.ts"
```
Comment (ryanleecode) [inline examples/smart_contract.ts:3]: what is the reason for this change?

Comment (ryanleecode) [reply examples/smart_contract.ts:3]: capi --> localhost that is

Comment (harrysolovay) [reply examples/smart_contract.ts:3]: Changed the import map example scope mapping of `capi/` to `capi/mod.ts` bc it was causing the frustrating import rewrite suggestions. This meant that `capi/frame...` became invalid. That being said, closing this out.

**Comment by harrysolovay** [comment]
Keeping the import map separate is more elegant. Closing after all.
